pennylane/pauli/pauli_arithmetic.py

```diff
@@ -166,7 +166,6 @@ class PauliWord(dict):
     -1j * Z(1) @ Z(2) @ X(0)
 
     We can multiply scalars to Pauli words or add/subtract them, resulting in a :class:`~PauliSentence` instance.
-
```

Comment:
small white space change that I think needs to be resolved?

Reply:
Yep thanks!

Reply:
Yep! Good catch, also fixed some other rendering issues that snuck in